repos/spack_repo/builtin/packages/py_apache_tvm_ffi/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyApacheTvmFfi(PythonPackage):
+    """Open ABI and FFI for Machine Learning Systems"""
+
+    homepage = "https://github.com/apache/tvm-ffi"
+    pypi = "apache-tvm-ffi/apache_tvm_ffi-0.1.3.tar.gz"
+    git = homepage + ".git"
+
+    license("Apache-2.0")
+
+    version("main", branch="main")
+    version("0.1.3", sha256="d33f0bc0d028cddf321d69724c916504272a7f03dfc1d8e507d9d0f88b6f7cbf")
+    version("0.1.2", sha256="ba45ebf98bab436442f3ee34c8b9c69e00797ae3529ea3df37a56aa7aa479cf2")
```

Comment:
How is this hash calculated. I'm getting `91f6e4e38572f7ce78c6df810cc16bdd1283fd925010b0e503697934d58bb7e7`.